public/index.html

```diff
@@ -78,6 +78,28 @@ <h2>PayJoin here to open these channels</h2>
 					<a href="" id="bip21"></a>
 					<p>Please use <a href="https://en.bitcoin.it/wiki/PayJoin_adoption" target="_blank">a wallet that supports</a> BIP 78 <a href="https://bitcoinmagazine.com/culture/blockchain-analysis-about-get-harder-p2ep-enters-testing-phase" target="blank">P2EP</a> PayJoins!</p>
 				</output>
+				<br>
+				<br>
+			</fieldset>
+			<fieldset id="getinbound-wrapper">
```

Comment:
It has nice styling

I'd rather not try replicate the style as a `div` class because then we will have mismatches if we change template.
Does something bad happen if reusing CSS classes for other purposes? It keeps our style lean

Reply:
Moving into new empty form